### ISSUE TITLE
Fix user avatars resizing and being oval, as well as text resizing when time is ticking

### ISSUE THREAD

**Issue by thehowl** [gnolang/gnochess#154]
[Screencast from 2023-09-25 20-47-44.webm](https://github.com/gnolang/gnochess/assets/4681308/8f2bbab0-12e2-4983-968c-b1cbff9c9279)

**Comment by alexiscolin** [comment]
@thehowl Yes because your window screen is in a weird responsive. Should be right in regular screen. But yeah not optimal. I'll update it very soon.
